### PR TITLE
Add custom themes wiki page

### PR DESCRIPTION
**Pull request by fun840**
At the moment there's only a section for installation, but in the future after it's better supported, there could also be a section for theme creation.

### CODE REVIEW

**Comment by tomodachi94** [comment]
@fun840 shouldn't nonstandard themes be put into another directory in `themes` instead of overwriting `custom`?

**Comment by fun840** [comment]
> @fun840 shouldn't nonstandard themes be put into another directory in `themes` instead of overwriting `custom`?

In the future that's how it will work, but at the moment you can only add custom themes by replacing the files in `custom`. Any other directories / themes won't be detected.

**Comment by tomodachi94** [comment]
> In the future that's how it will work, but at the moment you can only add custom themes by replacing the files in `custom`. Any other directories / themes won't be detected.

Ah alright.

File: src/wiki/help-pages/Custom-themes.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- Windows: `%APPDATA%\Prism Launcher`
+- Mac: `~/Library/Application Support/PrismLauncher`
+- Linux: `~/.local/share/PrismLauncher`
+  - Flatpak: `~/.var/app/org.prismlauncher.PrismLauncher/data/PrismLauncher`
+- Portable: In the PrismLauncher folder
+
+Once you're in the PrismLauncher folder, enter the `themes` folder, then enter the `custom` folder. Inside this folder, replace `theme.json` and `themeStyle.css` with the custom theme files of your choice.
+
+**Note:** This will overwrite the pre-existing files!
```
Comment (tomodachi94) [inline src/wiki/help-pages/Custom-themes.md:23]: Maybe add a suggestion to copy the files somewhere else before overwriting them?

**Review by tomodachi94 — APPROVED** [review]

**Pull request closed without merging**